parl/remote/tests/cluster_test.py

```diff
@@ -95,7 +95,7 @@ def test_actor_exception(self):
         except:
             pass
         actor2 = Actor()
-        time.sleep(30)
+        time.sleep(50)
```

Comment:
We do not suggest a long sleep any more, as it may cause longer time consumption or timeout error.  
Please use 10 testes here, each with a test and 10 seconds sleep.